examples/node/dynamic_codegen/route_guide/route_guide_server.js

```diff
@@ -23,7 +23,16 @@ var parseArgs = require('minimist');
 var path = require('path');
 var _ = require('lodash');
 var grpc = require('grpc');
-var routeguide = grpc.load(PROTO_PATH).routeguide;
+var protoLoader = require('@grpc/proto-loader');
+var packageDefinition = protoLoader.loadSync(
+    PROTO_PATH,
+    {keepCase: true,
+     longs: String,
+     enums: String,
+     defaults: true,
+     oneofs: true
+    });
+var routeguide = grpc.load(packageDefinition).routeguide;
```

Comment:
`var routeguide = grpc.loadPackageDefinition(packageDefinition).routeguide;`

Reply:
Thank you